CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unrealeased]
 
+## [2.24.1] - 2023-08-30
+
+### Fixed
+
+- Fix django templates using course-glimpse-list block
```

Comment:
You may need to warn people in UPGRADE.md in case of overrides.

Reply:
I will add it during the cherry pick